src/rules/function-url-quotes/__tests__/index.js

```diff
@@ -145,6 +145,10 @@ testRule(rule, {
     code: "@font-face { font-family: 'foo'; src: url($variable + \"foo.ttf\" + $variable); }",
   }, {
     code: "a { background: url(\"/images/my_image@2x.png\") }",
+  }, {
+    code: "@-moz-document url-prefix() {}",
+  }, {
+    code: "a { background: url() }",
```

Comment:
@Nitive because you can use `url("")` -> request to this page, in backend you can checking what need (based on header in request) and return image or some another stuff, and it is right. Also it is valid CSS, see spec parsing in https://url.spec.whatwg.org/#url-parsing and trying in https://jigsaw.w3.org/css-validator/validator
```css
a {
  background: url("");
  background: url('');
}
```
Rule should always check quotes exist or not